debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -447,6 +448,15 @@ protected void doStart() {
         }
     }
 
+    static AuthenticationException sanitizeAuthenticationException(AuthenticationException e) {
+        String sanitizedError = e.getMessage()
+                .replaceAll("[^\\P{Cc}\t\r\n]", "");
+        AuthenticationException sanitized = new AuthenticationException(sanitizedError,
+                e.getErrorCode(), e.getSqlState());
+        sanitized.setStackTrace(e.getStackTrace());
```

Comment:
Can we assume the unrecognized control characters only appear in error message but not stack trace?